codes/quantum/qubits/surface/triangle.yml

```diff
@@ -0,0 +1,56 @@
+    - 'The CAT states (CS) initialization and measurement method uses a row of \(d\) ancilla qubits along some edge of a triangle code with distance \(d\) to create and verify a CAT state that is to measure the logical operator along the same edge. Creating this CAT state takes \(O(d)\) time steps. To relaibly measure the logical state, the CAT state must be measured \(O(d)\) times resuling in \(O(d^2)\) time for logical measurement. Initialization is a similar procedure that requires \(O(d^2)\) time for logical operator measurements that occur \(O(d)\) times as well as \(O(d^2)\) time to project the code onto a logical state \cite{doi:10.22331/q-2017-04-25-2}.'
+
+  transversal_gates: 'Triangle codes have transversal single-qubit Clifford gates of order \(3\) like Clifford gate \(\bar{SH}\) that performs \(H\) on all qubit positions \(x > 0\) or at \(x = 0\) and \(z > 0\) and then perform \(SH\) on the central qubit. Note that qubits are places on integer lattics in the \(xy-, yz-,\) and \(xz-\) planes \cite{doi:10.22331/q-2017-04-25-2}.'
+
+  #general_gates:
+
+  decoders:
+    - 'For decoding a single layer of the triangle code, a decoding graph is laid on the code. Nodes of the graph are located at each stabilizer (center of the triangle graph) and has red or blue edges where red associates with \(X\) errors and blue with \(Z\) errors. To take into account any errors from measuring the error syndrome, a three-dimensional stack of the decoding graphs is laid ontop of the code with vertical edges connecting to qubits between layers \cite{doi:10.22331/q-2017-04-25-2}.'
+
+  fault_tolerance:
+    - 'The symmetry of triangle codes allows for fault-tolerant measurement and initialization in any Pauli basis \cite{doi:10.22331/q-2017-04-25-2}.'
```

Comment:
"encoding in any logical Pauli basis" (at least I think this is our convention of what you mean)